packages/cli/src/cli.ts

```diff
@@ -53,14 +53,33 @@ export async function cli(args: string[] = process.argv) {
           .demandCommand();
       },
       handler: (_args) => {},
+    })
+    .fail(async (msg, err, yargs) => {
```

Comment:
In this case I think we should be careful with `yargs` as we're shadowing the import which could cause confusion maybe `argv` would be better